.npmrc

```
@@ -1,3 +1,3 @@
-registry=http://registry.npmjs.com/
+registry=http://registry.npmjs.org/
```

Please make this https as well
```suggestion
registry=https://registry.npmjs.org/
```